definitions/tools/filter_vcf_depth.cwl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "filter variants at sites below a given sequence depth in each sample"
+requirements:
+    - class: DockerRequirement
+      dockerPull: mgibio/depth-filter:0.1
+    - class: ResourceRequirement
+      ramMin: 4000
+baseCommand: ["/opt/conda/bin/python3","/usr/bin/depth_filter.py"]
+arguments: 
+    ["TUMOR,NORMAL", {valueFrom: "$(runtime.outdir)/depth_filtered.vcf"}]
```

Comment:
It might make sense to parameterize the sample names in the tool and then specify this value in workflows that call it. Something like this:
https://github.com/genome/analysis-workflows/blob/60d8a9e6c5f571ec9b37f10290a1f4613013f3e1/definitions/pipelines/detect_variants.cwl#L279-L280

Reply:
Good call - added that, and pushed the dv changes that I forgot to commit last night